example.env

```diff
@@ -0,0 +1,6 @@
+# This is an environment file for configuring dynamic values.
+E2E_BASE_URL=https://openmrs-refapp.globalhealthapp.net/openmrs/
```

Comment:
can we leave this open so that any implementation add env variables of choice

Reply:
The base URL should be localhost url by default as tests should run against local changes. 

Also please include a step to the GH workflow to copy environment variables from the example file:

https://github.com/openmrs/openmrs-esm-patient-chart/blob/4c67e0dc6550ca77373c2cbd8a257dd750478908/.github/workflows/e2e.yml#L23